tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/table/TableTest.java

```diff
@@ -483,6 +490,11 @@ public void testXmlWithNull() {
                     + "<double_type>0.0</double_type><boolean_type>false</boolean_type>"
                     + "<string_type xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:nil=\"true\">"
                     + "</string_type></result></results>";
+        } else if (dbType == MYSQL) {
```

Comment:
What is the difference between this and Postgres one?

Reply:
No difference it seems. Will merge the two.